tests/psa-client-server/psasim/src/psa_sim_generate.pl

```diff
@@ -84,9 +95,6 @@
 # It's possible that a production version might not need both type and ctypename;
 # that was done for convenience and future-proofing during development.
 
-# We'll do psa_crypto_init() first
-put_crypto_init_first(\@functions);
```

Comment:
Then we won't need `put_crypto_init_first` anymore and the definition can be deleted from the end of the file.